R/db-sql.R

```diff
@@ -172,17 +172,32 @@ sql_query_save.DBIConnection <- function(con, sql, name, temporary = TRUE, ...)
 }
 #' @export
 #' @rdname db-sql
-sql_query_wrap <- function(con, from, name = unique_subquery_name(), ...) {
+sql_query_wrap <- function(con, from, name = unique_subquery_name(), ..., lvl = 0) {
   UseMethod("sql_query_wrap")
 }
 #' @export
-sql_query_wrap.DBIConnection <- function(con, from, name = unique_subquery_name(), ...) {
+sql_query_wrap.DBIConnection <- function(con, from, name = unique_subquery_name(), ..., lvl = 0) {
   if (is.ident(from)) {
     setNames(from, name)
   } else if (is.schema(from)) {
     setNames(as.sql(from, con), name)
   } else {
-    build_sql("(", from, ") ", ident(name %||% unique_subquery_name()), con = con)
+    ident_name <- ident(name %||% unique_subquery_name())
+    build_sql(ident_subquery(from, con, lvl), " ", ident_name, con = con)
+  }
+}
+
+ident_subquery <- function(from, con, lvl) {
```

Comment:
I think we'll need to export this for other packages to use?

Reply:
👍  Renamed to `sql_indent_subquery()` and exported.